base/acme/conf/database/postgresql/statements.conf

```diff
@@ -0,0 +1,166 @@
+getNonce=\
+SELECT \
+    "expires" \
+FROM \
+    "nonces" \
+WHERE \
+    "value" = ?
+
+addNonce=\
+INSERT INTO \
+    "nonces" ("value", "expires") \
+VALUES \
+    (?, ?)
+
+removeNonce=\
+DELETE FROM \
+    "nonces" \
+WHERE \
+    "value" = ?
+
+removeExpiredNonces=\
+SELECT \
```

Comment:
`DELETE FROM`? Or does this `getExpiredNonces` so that `removeNonce` can be called on each one? Seems like letting SQL do it in bulk is better policy. 

In the future (with more configuration options), we should add a cleanup timer to the database configuration.

Reply:
Fixed. It's changed to `getExpiredNonces`, which is called by `removeExpiredNonces`, then the expired nonces are removed individually with `removeNonce`. Yes, we can certainly improve this.